docs/cloud/managing-permissions.md

```diff
@@ -0,0 +1,83 @@
+# Managing permissions
```

Comment:
@brandonSc - can you verify that the satellite-related permissions are accurate?

@mikejholly, @r-LaForge & @mirho - can you verify that the Earthly CI permissions are accurate?

Reply:
Right now the Satellites code isn't aware of the permissions. i.e. if you're a member of the org, you can do anything.

I think we can implement the permissions as you described now though. We just didn't have a good way to access them when we initially wrote satellites.